testsuite/feelmodels/test_modelproperties.cpp

```diff
@@ -21,28 +21,32 @@
  License along with this library; if not, write to the Free Software
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
+#define BOOST_TEST_MODULE model properties testsuite
+
+#include <testsuite.hpp>
+
 #include <feel/feelcore/environment.hpp>
 #include <feel/feelmodels/modelproperties.hpp>
 #include <feel/feeldiscr/pch.hpp>
 #include <feel/feelfilters/loadmesh.hpp>
 #include <feel/feelvf/vf.hpp>
 
-int main( int argc, char** argv )
+FEELPP_ENVIRONMENT_NO_OPTIONS
+
+BOOST_AUTO_TEST_SUITE( modelproperties )
+
+BOOST_AUTO_TEST_CASE( test_materials )
 {
     using namespace Feel;
-    Environment env( _argc=argc, _argv=argv );
-
     auto mesh = loadMesh(new Mesh<Simplex<3> >);
     auto Xh = Pch<1>(mesh);
     auto g = Xh->element();
     auto d = Xh->element();
 
-    ModelProperties model_props( "test.feelpp" );
-
+    ModelProperties model_props( Environment::expand("$testsuite_srcdir/feelmodels/test.feelpp" ) );
     auto mats = model_props.materials();
```

Comment:
Use an option instead hardcoded the path (because it doesn't work when testsuite  is installed)
The cfg will contain  ```myoption=$cfgdir/test.feelpp```